pages/StartPage/index.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import StartButton from "../../components/StartPage/StartButton";
+import HeaderStartPage from "../../components/StartPage/Header";
+import StartCard from "../../components/StartPage/QuizCardStart";
+
+export default function StartPage() {
+  return (
+    <>
+      <StyledForm>
```

Comment:
There is no need for this `form`, is it? There are no input fields nor a submit button.

Reply:
change it to section